testsuite/MathMLToDisplay/Characters/issue898-2.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html class="reftest-wait">
+  <head>
+    <title>SVG output of space characters</title>
+    <!-- Copyright (c) 2015 MathJax Consortium
+         License: Apache License 2.0 -->
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
+    <script type="text/javascript" src="../../header.js"></script>
+    <script type="text/javascript" src="../../scriptTests.js"></script>
+    <script type="text/javascript">
+      function getTestCases() {
+        var svgs = document.getElementsByTagName('svg');
```

Comment:
Since there is only one SVG element, might as well use

    var svg = document.getElementsByTagName('svg')[0];
    var check1 = (svg.innerHTML.indexOf('font-style=""') > -1);
    var check2 = (svg.innerHTML.indexOf('font-weight=""') > -1);

rather than use the loops for just one element.